LibraBFT/Base/KVMap.agda

```diff
@@ -59,8 +59,16 @@ module LibraBFT.Base.KVMap  where
                   → lookup k kvm ≢ nothing
                   → KVMap Key Val
 
+   kvm-insert-Haskell
+                  : (k : Key)(v : Val)(kvm : KVMap Key Val)
+                  → KVMap Key Val
+
+   kvm-member     : (k : Key) (kvm : KVMap Key Val)
+                  → Bool
+
    kvm-size       : KVMap Key Val → ℕ
    -- TODO-1: add properties relating kvm-toList to empty, kvm-insert and kvm-update
+   kvm-keys       : KVMap Key Val → List Key
```

Comment:
I suggest defining `kvm-keys` as `map proj1 . kvm-toList`

Reply:
I am going to deal with KVMap changes in an independent pull request.